src/SpectatorView.Unity/Assets/SpectatorView/Scripts/StateSynchronization/AssetCache.cs

```diff
@@ -178,83 +220,158 @@ public static IEnumerable<T> EnumerateAllComponentsInScenesAndPrefabs<T>()
         }
     }
 
-    internal abstract class AssetCache<TAssetEntry, TAsset> : AssetCache, IAssetCache
-        where TAssetEntry : AssetCacheEntry<TAsset>, new()
+    internal abstract class AssetCache<TAsset> : AssetCache, IAssetCache
         where TAsset : UnityEngine.Object
     {
-        [SerializeField]
-        private TAssetEntry[] assets = null;
+        const string assetContentExtension = ".asset";

```

Comment:
>assetContentExtension [](start = 21, length = 21)

I think we should stick to Pascal case: 

[https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions)
[https://blogs.msdn.microsoft.com/mschray/2013/05/06/practical-naming-standards-for-c/](https://blogs.msdn.microsoft.com/mschray/2013/05/06/practical-naming-standards-for-c/)

So AssetContentExtensions.

Especially for constants

Reply:
sounds good.

---
In reply to: [355037299](https://github.com/microsoft/MixedReality-SpectatorView/pull/295#discussion_r355037299) [](ancestors = 355037299)